Makefile

```diff
@@ -44,13 +44,11 @@ vet: ## Verifies `go vet` passes.
 .PHONY: cover
 cover: ## Runs go test with coverage.
 	@echo "" > coverage.txt
-	@for d in $(shell $(GO) list ./... | grep -v vendor); do \
-		$(V)GO111MODULE=on $(GO) test -race -coverprofile=profile.out -covermode=atomic "$$d"; \
-		if [ -f profile.out ]; then \
-			cat profile.out >> coverage.txt; \
-			rm profile.out; \
-		fi; \
-	done;
+	$(V)GO111MODULE=on $(GO) test -race -coverprofile=profile.out -covermode=atomic ./...; \
+	if [ -f profile.out ]; then \
+		cat profile.out >> coverage.txt; \
+		rm profile.out; \
+	fi;
```

Comment:
I'm really not sure why lines 48-51 are necessary. Why not just remove the `@echo "" > coverage.txt` line and do:

```
$(V)GO111MODULE=on $(GO) test -race -coverprofile= coverage.txt -covermode=atomic ./...
```

Reply:
remaining bits from the copy/paste template of the plugin example. my bad